sdk/export/metric/aggregation/aggregation.go

```diff
@@ -27,11 +28,17 @@ type Sum = aggregation.Sum
 // Deprecated: use module "go.opentelemetry.io/otel/sdk/metric/export/aggregation"
 type Count = aggregation.Count
 
-// Deprecated: use module "go.opentelemetry.io/otel/sdk/metric/export/aggregation"
-type Min = aggregation.Min
+// Deprecated: Will be removed soon.
+type Min interface {
+	Aggregation
+	Min() (number.Number, error)
+}
 
-// Deprecated: use module "go.opentelemetry.io/otel/sdk/metric/export/aggregation"
-type Max = aggregation.Max
+// Deprecated: Will be removed soon.
+type Max interface {
+	Aggregation
+	Max() (number.Number, error)
+}
```

Comment:
I kept them in the deprecated package, but removed them from the new package since they are not used. They will be removed with the whole deprecated package.